store/manager.go

```diff
@@ -24,45 +27,53 @@ type Manager struct {
 
 	s3 common.PrecomputedKeyStore // for op keccak256 commitment
 	// For op generic commitments & standard commitments
-	eigenda   common.GeneratedKeyStore // v0 da commitment version
-	eigendaV2 common.GeneratedKeyStore // v1 da commitment version
-	writeV2   bool                     // write blobs to EigenDAV2 backend
+	eigenda    common.GeneratedKeyStore // v0 da commitment version
+	eigendaV2  common.GeneratedKeyStore // v1 da commitment version
+	disperseV2 *atomic.Bool             // disperse blobs to EigenDAV2 backend
```

Comment:
Nit - does this actually need to be atomic? Iiuc there's isn't a current test case which distributes client load in parallel when switching the version flag

Reply:
needs to be atomic to prevent race condition from proxy reading it, while test writing to it. Prob innocuous race condition but still better to be race free

Reply:
Making it atomic also keeps the door open for exposing the flag to users, if there would ever be a use case. Maybe not likely, but the atomicity cost is negligible at the rates we'll see here.